src/app/modules/funds/page/fund-details/fund-details.component.html

```diff
@@ -1,10 +1,28 @@
 <mat-card>
   <mat-card-title>
-    <div class="fund-title-container">
+    <div class="fund-title-container" (clickOutside)="closeEditFundName()">
       <span class="fund-title-container__btn" (click)="back()">
         <mat-icon color="primary">arrow_back</mat-icon>
       </span>
-      <h2 class="fund-title">{{fund?.name || 'Fund name'}}</h2>
+      <div *ngIf="isEditFundName; else fundNameBlock">
+        <mat-form-field appearance="legacy" class="fund-title-container__input">
+          <input matInput placeholder="Fund name" [formControl]="fundName" required>
+          <mat-error *ngIf="fundName.invalid">You must enter fund name</mat-error>
```

Comment:
**You must...** sounds like strong obligation to do something, but not a recommendation. Could you, please, change to "Please, enter fund name" or something like that.